sympy/core/power.py

```diff
@@ -1302,7 +1302,14 @@ def _eval_as_leading_term(self, x):
         from sympy import exp, log
         if not self.exp.has(x):
             return self.func(self.base.as_leading_term(x), self.exp)
-        return exp(self.exp * log(self.base)).as_leading_term(x)
```

Comment:
Why not just `return self`?  The concept of a leading term for a sum series seems not to make sense when the variable occurs in both the base and the exponent.